README.md

```diff
@@ -67,22 +67,9 @@ Every element has been normalized/reset to a simple visually consistent style th
 
 More customization examples and best practices coming soon.
 
-### Using classes instead of element selectors
+### Using classes to style
 
-Although we recommend thinking of this plugin as a "form reset" rather than a collection of form component styles, in some cases our default approach may be too heavy-handed, especially when integrating this plugin into existing projects.
-
-For situations where the default strategy doesn't work well with your project, you can use the `class` strategy to make all form styling _opt-in_ instead of applied globally:
-
-```js
-// tailwind.config.js
-plugins: [
- require("@tailwindcss/forms")({
-   strategy: 'class',
- }),
-],
-```
-
-When using the `class` strategy, form elements do not receive any reset styles by default, and reset styles are added per element using a new set of `form-*` classes generated by the plugin:
+In addition to the base styles to automatically style each form element we also generate classes that you can use explicitly. This allows you to style form elements by default and have the ability to style non-form things to more closely match those form elements.
```

Comment:
Possible rewording...
```suggestion
In addition to the global styles, we also generate a set of corresponding classes which can be used to explicitly apply the form styles to an element. This can be useful in situations where you need to make a non-form element, such as a `<div>`, look like a form element.
```

Reply:
Oh that's much, much better. 👍